tests/unit/test_temporal_client.py

```
@@ -0,0 +1,97 @@
+        self.env_patcher = patch.dict("os.environ", self.default_env)
+        self.env_patcher.start()
+        self.client = TemporalClient()
+
+    def tearDown(self):
+        """Clean up test environment after each test"""
+        self.env_patcher.stop()
+
+    def test_initialization(self):
+        """Test class initialization and load_dotenv call"""
+        with patch("utils.temporal_client.load_dotenv") as mock_load_dotenv:
+            client = TemporalClient()
```

Ruff (0.7.0)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)